src/components/learner-credit-management/BudgetDetailActivityTabContents.jsx

```diff
@@ -1,47 +1,70 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { useParams } from 'react-router-dom';
+import { Stack } from '@edx/paragon';
 
-import LearnerCreditAllocationTable from './LearnerCreditAllocationTable';
-import { useOfferRedemptions, isUUID } from './data';
+import BudgetDetailRedemptions from './BudgetDetailRedemptions';
+import BudgetDetailAssignments from './BudgetDetailAssignments';
+import {
+  useOfferRedemptions,
+  useBudgetContentAssignments,
+  useBudgetId,
+  useSubsidyAccessPolicy,
+} from './data';
 
 const BudgetDetailActivityTabContents = ({
   enterpriseUUID,
-  enterpriseSlug,
-  enableLearnerPortal,
+  enterpriseFeatures,
 }) => {
-  const { budgetId } = useParams();
-  const enterpriseOfferId = isUUID(budgetId) ? null : budgetId;
-  const subsidyAccessPolicyId = isUUID(budgetId) ? budgetId : null;
+  const { enterpriseOfferId, subsidyAccessPolicyId } = useBudgetId();
+  const {
+    data: subsidyAccessPolicy,
+  } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+
+  const isTopDownAssignmentEnabled = enterpriseFeatures?.topDownAssignmentRealTimeLcm;
+
   const {
     isLoading: isLoadingOfferRedemptions,
     offerRedemptions,
     fetchOfferRedemptions,
   } = useOfferRedemptions(enterpriseUUID, enterpriseOfferId, subsidyAccessPolicyId);
 
+  const {
+    isLoading: isLoadingContentAssignments,
+    contentAssignments,
+    fetchContentAssignments,
+  } = useBudgetContentAssignments({
+    assignmentConfigurationUUID: subsidyAccessPolicy?.assignmentConfiguration?.uuid,
```

Comment:
[question] can you share some insight on `assignmentConfiguration`. How is that configuration set in the BE? Just looking through the staged data, it looks like this value is always `null`.

Reply:
There is a new SubsidyAccessPolicy type for assignable learner credit. When you create a new one ([example](https://enterprise-access-internal.stage.edx.org/admin/subsidy_access_policy/assignedlearnercreditaccesspolicy/add/)), there is an editable field to add/edit an `AssignmentConfiguration`. From there, you can manually create assignments for the `AssignmentConfiguration` [here](https://enterprise-access-internal.stage.edx.org/admin/content_assignments/learnercontentassignment/).

<img width="1147" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/d52902b8-590a-4267-b626-9f029892a866">